EIPS/Increase block rewards to 5 ETH.md

```diff
@@ -0,0 +1,56 @@
+
+Therefore, the ultimate end goal for this EIP is to give POW miners more incentive to switch to POS once ETH 2.0 is fully implemented since the transition will take a few years to complete and during that time, they will be incentivised to hold on to the tokens instead of selling it straightaway in order to prepare to be a validator for ETH 2.0, reducing the selling pressure on ETH and increasing it's value in the long run. A side effect of miners staying on Ethereum is that network security will be assured during the transition period.
+
+## Specification
+#### Adjust Block, Uncle, and Nephew rewards
+To ensure a constant Ether issuance, adjust the block reward to `new_block_reward`, where
+
+    new_block_reward = 5_000_000_000_000_000_000 if block.number >= FORK_BLKNUM else block.reward
+
+(5E18 wei, or 5,000,000,000,000,000,000 wei, or 5 ETH).
+
+Analogue, if an uncle is included in a block for `block.number >= FORK_BLKNUM` such that `block.number - uncle.number = k`, the uncle reward is
+
+    new_uncle_reward = (8 - k) * new_block_reward / 8
+
+This is the existing formula for uncle rewards, simply adjusted with `new_block_reward`.
+
+The nephew reward for `block.number >= FORK_BLKNUM` is
+
+    new_nephew_reward = new_block_reward / 32
+
+This is the existing formula for nephew rewards, simply adjusted with `new_block_reward`.
+
+## Rationale
+I have decided to go with a 5 ETH base reward as i believe that it is a good middle ground between wanting to prevent too high of an inflation rate (10.4% per annum for the first year if we go with 5 ETH) and the desires to convert as many POW miners as possible into POS validators by making it easier to amass the required ETH needed through POW mining.
```

Comment:
Nitpick: Consider rephrasing this from "I have decided" to a statement of fact.  Ideally, EIPs are written for a target audience of future readers after the EIP has been accepted and implemented somewhere.

```suggestion
A 5 ETH base reward was chosen as a middle ground between wanting to prevent too high of an inflation rate (10.4% per annum for the first year at 5 ETH per block) and converting as many POW miners as possible into POS validators by making it easier to amass the required ETH needed through POW mining.
```